chroma-manager/tests/integration/shared_storage_configuration/test_conf_params.py

```diff
@@ -93,8 +93,7 @@ def test_update_conf_params(self):
         self.hosts = self.add_hosts(host_addresses)
         self.configure_power_control(host_addresses)
 
-        volumes = self.get_usable_volumes()
-        self.assertGreaterEqual(len(volumes), 4)
```

Comment:
Why don't we assert here anymore?

Reply:
assert is implicit inside wait_usable_volumes(4)